python/cuda_parallel/pyproject.toml

```diff
@@ -19,8 +19,10 @@ requires-python = ">=3.9"
 dependencies = [
   "cuda-cccl",
   "numba>=0.60.0",
-  "cuda-python==12.*",
+  "cuda-python==12.9.0",
```

Comment:
Should we use `>=` rather than `==` here?

Reply:
I suggested it originally, but changed my mind because we are using the private api ftm, and it is likely that we'd turn it public in the next patch release, which could break cuda.parallel.

Reply:
Thanks. Then maybe let's note that the pin is due to using the experimental `path_finder` API, and that once we switch to a stable API we can make the pinning more lax

Reply:
@leofang How about `cuda-python>=12.9.0,<13.0.0a0`?

Reply:
commit a1ef0525f244812c85a390d67640d2039ec4d416